Tests/LibGfx/TestImageDecoder.cpp

```diff
@@ -101,6 +101,23 @@ TEST_CASE(test_jpeg_sof0_one_scan)
     EXPECT(frame.duration == 0);
 }
 
+TEST_CASE(test_jpeg_sof0_several_scans)
+{
+    auto file = MUST(Core::MappedFile::map(TEST_INPUT("several_scans.jpg"sv)));
+    EXPECT(Gfx::JPEGImageDecoderPlugin::sniff(file->bytes()));
+    auto plugin_decoder = MUST(Gfx::JPEGImageDecoderPlugin::create(file->bytes()));
+    EXPECT(plugin_decoder->initialize());
+
+    EXPECT(plugin_decoder->frame_count());
+    EXPECT(!plugin_decoder->is_animated());
+    EXPECT(!plugin_decoder->loop_count());
+
+    EXPECT(!plugin_decoder->frame(0).is_error());
+
+    auto frame = plugin_decoder->frame(0).release_value_but_fixme_should_propagate_errors();
```

Comment:
`auto frame = MUST(plugin_decoder->frame(0));`

…but I thought you wanted to check the size, not get the frame? (getting the frame is fine too, also serves in exercising the new code I suppose)

Reply:
You were the one proposing testing the size, I don't think either of it has value.